src/lib/utilities/deleteAllChats.ts

```diff
@@ -0,0 +1,77 @@
+export const deleteAllChats = (): Error | void => {
+    try {
+        const chatHistory = document.querySelector(
+            "div.group\\/sidebar> div.flex.flex-col.gap-2.text-token-text-primary.text-sm.false.mt-5.pb-2",
+        );
+
+        if (!chatHistory) throw new Error("No chat history found");
+
+        const profileButton = document.querySelector(
+            '[aria-label="Open Profile Menu"]',
+        ) as HTMLButtonElement;
+
+        if (!profileButton) throw new Error("Profile button not found!");
+
+        profileButton.dispatchEvent(
+            new PointerEvent("pointerdown", {
+                bubbles: true,
+                cancelable: true,
+                view: window,
+                button: 0,
+                pointerType: "mouse",
+            }),
+        );
+
+        profileButton.dispatchEvent(
+            new PointerEvent("pointerup", {
+                bubbles: true,
+                cancelable: true,
+                view: window,
+                button: 0,
+                pointerType: "mouse",
+            }),
+        );
+
+        const settingsMenuItem = document.querySelector(
```

Comment:
add an inline comment here that this querySelector may need to be updated if the domain updates their html